src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/FlightEnabledAnalysis/FlightEnabledAnalysis.FlightEnabledDataFlowOperationVisitor.cs

```diff
@@ -0,0 +1,211 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.PointsToAnalysis;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.FlightEnabledAnalysis
+{
+    using FlightEnabledAnalysisData = DictionaryAnalysisData<AnalysisEntity, FlightEnabledAbstractValue>;
+    using FlightEnabledAnalysisDomain = MapAbstractDomain<AnalysisEntity, FlightEnabledAbstractValue>;
+    using FlightEnabledAnalysisResult = DataFlowAnalysisResult<FlightEnabledBlockAnalysisResult, FlightEnabledAbstractValue>;
+
+    internal partial class FlightEnabledAnalysis : ForwardDataFlowAnalysis<FlightEnabledAnalysisData, FlightEnabledAnalysisContext, FlightEnabledAnalysisResult, FlightEnabledBlockAnalysisResult, FlightEnabledAbstractValue>
+    {
+        /// <summary>
+        /// Operation visitor to flow the FlightEnabled values across a given statement in a basic block.
+        /// </summary>
+        private sealed class FlightEnabledDataFlowOperationVisitor
+            : AnalysisEntityDataFlowOperationVisitor<FlightEnabledAnalysisData, FlightEnabledAnalysisContext, FlightEnabledAnalysisResult, FlightEnabledAbstractValue>
+        {
+            private readonly AnalysisEntity _globalEntity;
```

Comment:
This is the global entity storing CFG wide state, which gets updated for every visited operation in the visitor.

Reply:
```suggestion
            // This is the global entity storing CFG wide state, which gets updated for every visited operation in the visitor.
            private readonly AnalysisEntity _globalEntity;
```